classes/02class/exercises/c04-iam01/danielvca/README.md

```diff
@@ -0,0 +1,15 @@
+﻿
+# c04-iam01
+
+## Command Execution Output
+
+- [cli_commands.txt](cli_commands.txt)
```

Comment:
can you please use the provided template to submit your questions? It helps a lot in our reviewing process to follow a standard